Model/Sales/Repository/PackageRepository.php

```diff
@@ -55,14 +142,176 @@ public function setWeightFromQuoteProducts($products)
             return $this;
         }
 
-        $this->setWeight(0);
+        $this->addWeight(0);
```

Comment:
Waarom?

Reply:
$this->addWeight(0) of $this->setWeight(0)  is helemaal niet nodig

Reply:
Done